homeassistant/components/esphome/strings.json

```
@@ -47,8 +47,8 @@
   },
   "issues": {
     "ble_firmware_outdated": {
-      "title": "Update {name} with ESPHome 2022.11.0 or later",
-      "description": "To improve Bluetooth reliability and performance, we highly recommend updating {name} with ESPHome 2022.11.0 or later."
+      "title": "Update {name} with ESPHome {version} or later",
+      "description": "To improve Bluetooth reliability and performance, we highly recommend updating {name} with ESPHome {version} or later. When updating to ESPHome {version}, it is recommended to use a serial cable instead of an over the air update to take advantage of the new partition scheme."
```

Maybe add "the device" to make it clear it is about updating the device itself and not, e.g., the add-on?

```suggestion
      "description": "To improve Bluetooth reliability and performance, we highly recommend updating {name} with ESPHome {version} or later. When updating the device to ESPHome {version}, it is recommended to use a serial cable instead of an over-the-air update to take advantage of the new partition scheme.",
```

---

Sounds good to me.  Lots of versions here, anything we can do to make it less confusing is a win 👍

---

Yeah, that is how I looked at it as well.